node/pkg/driver/node_utils.go

```diff
@@ -320,6 +328,73 @@ func (n NodeUtils) MakeFile(filePath string) error {
 	return nil
 }
 
+func (n NodeUtils) ExpandFilesystem(devicePath string, fsType string) error {
+	var cmd string
+	var args []string
+	if fsType == "ext4" {
+		cmd = "resize2fs"
+		args = []string{devicePath}
+	} else if fsType == "xfs" {
+		cmd = "xfs_growfs"
+		args = []string{"-d", devicePath}
+	} else {
+		logger.Warningf("Skipping resize of unsupported fsType: %v", fsType)
+		return nil
+	}
+
+	logger.Debugf("Resizing the device with fs_type = {%v}", fsType)
+	_, err := n.Executer.ExecuteWithTimeout(resizeFsTimeoutMilliseconds, cmd, args)
+	if err != nil {
+		logger.Errorf("Failed to resize filesystem, error: %v", err)
+	}
+	return nil
+}
+
+func (n NodeUtils) ExpandMpathDevice(mpathDevice string) error {
+	logger.Infof("ExpandMpathDevice: [%s] ", mpathDevice)
+	args := []string{"resize", "map", mpathDevice}
+	output, err := n.Executer.ExecuteWithTimeout(TimeOutMultipathdCmd, multipathdCmd, args)
+	if err != nil {
+		return fmt.Errorf("multipathd resize failed: %v\narguments: %v\nOutput: %s\n", err, args, string(output))
+	}
+	return nil
+}
+
+func (n NodeUtils) rescanPhysicalDevice(deviceName string) error {
```

Comment:
why would we want the first rename?
I did the second rename (though this confusion still exists in the project, e.g. `RemovePhysicalDevice`).

IMO the node.go has enough logic as it is, and I don't think it would ever need a single physical device rescan.

Reply:
the convention seems to be with first letter capital

Reply:
so why not combine them for single method that going trough all devices ?
(only RescanPhysicalDevices)

Reply:
I don't think that is a matter of convention, if a public function is getting large, it should be divided into smaller functions, and IMO combining them creates a function which is too long.

Reply:
ok, Im closing this conversation, hope it wont present any issue in UT, if it does it will handled then